frontend/integration-tests/tests/crd-extensions.scenario.ts

```diff
@@ -199,4 +199,91 @@ describe('CRD extensions', () => {
     });
   });
 
+  describe('ConsoleExternalLogLink CRD', () => {
+    const crd = 'ConsoleExternalLogLink';
+    const name = `${testName}-cell`;
+    const podName = `${testName}-pod`;
+    const cell = $(`[data-test-id=${name}]`);
+    const text = `${name} Logs`;
+    const namespaceFilter = '^openshift-';
+
+    it(`displays YAML editor for creating a new ${crd} instance`, async() => {
+      await browser.get(`${appHost}/k8s/cluster/customresourcedefinitions?name=${crd}`);
```

Comment:
I feel like we should have a utility function for creating things from YAML, so it's not repeated in so many places. But not for this PR.